infrastructure/docker/build/Dockerfile-tsb

```diff
@@ -0,0 +1,32 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM golang:1.13 as tsb-build
```

Comment:
Our repo no longer references Go versions in multiple places and instead downloads the version of Go specified by the [`GO_VERSION`](https://github.com/apache/trafficcontrol/blob/f8b34009c6/GO_VERSION) file at the root of the project working tree.

See #5095 for examples of modifying Dockerfiles with a `golang:1.##.#` base image to use `GO_VERSION` with a `debian:buster` base image.

Reply:
tsb does not necessarily run the same version of go as Traffic Control and go is not backward compatible between versions.

Reply:
Apache Traffic Control sticks to supported versions of Go. tsb should stick to being buildable using supported versions of Go, too.

Go 1.13 is not supported, and rather than upgrading multiple Dockerfiles throughout the repo whenever an older version of Go is no longer supported, using `GO_VERSION` saves us a lot of effort in the long run.

Reply:
Ok, I went ahead and switched this to a generic image. It's not super-likely to break and if it does, the ATC team will notice so we can open a PR to fix it. :D